lib/advisor/core/questions/questions.yml

```diff
@@ -1,21 +1,17 @@
 technical:
   - "How could this person become a better pair?"
-  - "How could the this person improve their code quality?"
+  - "how could the this person improve their code quality?"
```

Comment:
`How could this person` - you have an extra word `the`

Reply:
Thanks! I'll tune the questions themselves in a separate PR.